noxfiles/ci_nox.py

```diff
@@ -0,0 +1,202 @@
+"""Contains the nox sessions used during CI checks."""
+import nox
+from constants_nox import (
+    CI_ARGS,
+    COMPOSE_FILE,
+    IMAGE_NAME,
+    INTEGRATION_COMPOSE_FILE,
+    RUN,
+    RUN_NO_DEPS,
+    START_APP,
+    WITH_TEST_CONFIG,
+)
+from docker_nox import build_local_prod
+from utils_nox import teardown
+
+RUN_STATIC_ANALYSIS = (*RUN_NO_DEPS, "nox", "-s")
+
+
+@nox.session()
+def check_all(session: nox.Session) -> None:
```

Comment:
`pytest_external` is not included in with `check_all`. This may have been intentional since external collaborators can't run the external tests?